src/Driver/PgSQL/Statement.php

```diff
@@ -81,8 +81,15 @@ public function bindValue($param, $value, $type = ParameterType::STRING): bool
             throw UnknownParameter::new((string) $param);
         }
 
-        $this->parameters[$this->parameterMap[$param]]     = $value;
-        $this->parameterTypes[$this->parameterMap[$param]] = $type;
+        if ($type === ParameterType::BOOLEAN) {
+            // convert to boolean as value got transformed to int earlier
```

Comment:
Don't we just want to convert any value to a boolean here? This comment doesn't seem to provide any value.